apprentice_teacher.rb

```diff
@@ -26,15 +18,6 @@ def teach_stuff
     response
   end
 
-  def salary=(new_salary)
-    puts "This better be good!"
-    @salary = new_salary
-  end
-
-  def receive_raise(raise)
-    @salary += raise
-  end
-
   def set_performance_rating(rating)
```

Comment:
This is the same between teacher and apprentice teacher except for a single number.  How can you DRY that up?